socorro/signature/rules.py

```diff
@@ -525,13 +526,24 @@ def action(self, raw_crash, processed_crash, notes):
             # Recent crash reports added some irrelevant information at the
             # beginning of the abort message. We want to remove that and keep
             # just the actual abort message.
-            abort_message = abort_message.split('###!!! ABORT:', 1)[1].strip()
+            abort_message = abort_message.split('###!!! ABORT:', 1)[1]
 
         if ': file ' in abort_message:
             # Abort messages contain a file name and a line number. Since
             # those are very likely to change between builds, we want to
             # remove those parts from the signature.
-            abort_message = abort_message.split(': file ', 1)[0].strip()
+            abort_message = abort_message.split(': file ', 1)[0]
+
+        if 'unable to find a usable font' in abort_message:
+            # "unable to find a usable font" messages include a parenthesized localized message. We
+            # want to remove that. Bug #1385966
+            open_paren = abort_message.find('(')
+            if open_paren != -1:
+                end_paren = abort_message.rfind(')')
+                if end_paren != -1:
+                    abort_message = abort_message[:open_paren] + abort_message[end_paren + 1:]
+
+        abort_message = drop_unicode(abort_message).strip()
```

Comment:
+1. I was worried about dropping unicode before, based on an intuition that I don't understand all the possible error messages and where unicode may be important. After some consideration I think this is a good, general way to handle it, the specifics should be preserved in individual files, and we can address this later if it becomes a problem.